pilot/pkg/model/authorization.go

```diff
@@ -135,19 +141,30 @@ func (policy *AuthorizationPolicies) ListServiceRoleBindings(ns string) map[stri
 	return rolesAndBindings.Bindings
 }
 
-// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in the given namespace.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(ns string, workloadLabels labels.Collection) []Config {
+// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in root namespace and the config namespace.
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(configNamespace string,
+	workloadLabels labels.Collection) []Config {
 	if policy == nil {
 		return nil
 	}
 
-	var ret []Config
+	var namespaces []string
+	if policy.rootNamespace != "" {
```

Comment:
Here I'm following the same logic in all networking API and they doesn't have the default "istio-config" root namespace. Could you provide a link to the default "istio-config"? we should follow it everywhere in Pilot and also document it on istio.io

cc @rshriram @howardjohn @hzxuzhonghu

Reply:
The intent was root namespace would always have a value (istio-config) but I don't think this is the reality of pilot code. I think the default is now istio-system

Reply:
Thanks for the input John. @liminw How about let's not block this PR for this? I will file a separate issue to make usre all APIs (networking, authz) have the same behavior to fallback to use `istio-config`?

Reply:
Addressing it in a separate issue is fine with me. But I want to make sure that we do have a default root namespace. The root_namespace configuration in meshConfig is just used for the case that root namespace is set to a value that is different from the default. Root namespace is used for mesh level policies. It must always have a valid value, otherwise, there is no way to specify mesh policies.

Reply:
I agree with the default root namespace. I just think it should be addressed in the same way for all related APIs in Pilot to make sure we have a consistent behavior regarding the root namespace across APIs in Istio.